### PR TITLE
[Chronos] add evaluation function to benchmark tool.

### PR DESCRIPTION
**Pull request by NingWang5**
## Description

Add the "evaluation function" to chronos-benchmark-tool, which can provide accuracy caculation.

### 1. Why the change?

New features to perfect the chronos-benchmark-tool.

#### change
"-s/--stage" can choose from "train/latency/throughput/evaluation". 
#### new
"--metrics", evaluation metrics of a trained model, e.g. "mse"/"mae"
"--normalization", if to use normalization trick to alleviate distribution shift.

#### benchmark-chronos.py
Add the evalution function, revise argsparser and output function.

#### models.py
Add the "metrics" parameter to all forecaseters when initialization, and add "normalization" to TCN model.

#### utils_data.py
Add the validation data.

#### env_check.py
Remove redundant function calls, which have been called in benchmark-chronos.py.

### 4. How to test?

In script folder, running the following command:
`python3 benchmark-chronos.py -s evaluation -l 96 -o 720 --training_epochs 20`

### CODE REVIEW

File: python/chronos/script/benchmark-chronos.py [hunk 1]
```diff
@@ -171,7 +171,24 @@ def latency():
         records['openvino_percentile_latency'] = np.percentile(latency_vino, latency_percentile)
 
 
+def evaluation():
+    """
+    evaluate stage will record model accuracy.
+    """
+
+    forecaster.fit(train_loader, validation_data=val_loader, epochs=args.training_epochs, validation_mode="best_epoch")
```
Comment (TheaperDeng) [inline python/chronos/script/benchmark-chronos.py:179]: recently I found "best_epoch" might cause some issue like "val/loss" not found. Please pay attention if same issue happens to you

File: python/chronos/script/benchmark-chronos.py [hunk 2]
```diff
@@ -213,7 +236,7 @@ def result():
                         help=('model name, choose from tcn/lstm/seq2seq/nbeats/autoformer,'
                               ' default to "tcn".'))
     parser.add_argument('-s', '--stage', type=str, default='train', metavar='',
-                        help=('stage name, choose from train/latency/throughput,'
+                        help=('stage name, choose from train/latency/throughput/evaluation,'
```
[the remark this answers is not in the document]
Comment (NingWang5) [reply python/chronos/script/benchmark-chronos.py]: Has changed

File: python/chronos/script/benchmark-chronos.py (hunk 1 above)
Comment (NingWang5) [reply python/chronos/script/benchmark-chronos.py]: OK

**Review by TheaperDeng — APPROVED** [review]
LGTM
